src/Uno.UWP/Devices/Bluetooth/GenericAttributeProfile/GattDescriptorsResult.cs

```diff
@@ -7,6 +9,13 @@ public partial class GattDescriptorsResult
 		public  IReadOnlyList<GattDescriptor> Descriptors { get; internal set; }
 		public  byte? ProtocolError { get; internal set; }
 		public  GattCommunicationStatus Status { get; internal set; }
+
+		private GattDescriptorsResult(IReadOnlyList<GattDescriptor> param)
+		{
+			// dummy for Error CS8618  Non-nullable property 'Value' must contain a non-null value when exiting constructor. Consider declaring the property as nullable.
+			Descriptors = param;
```

Comment:
Let's create an empty list by default instead.

Reply:
This constructor probably would be used in 'real code'.
This object cannot be created as such (no public ctor), it can only be returned inside other structs/classes from Bluetooth implementation.